recipes/poliastro/run_test.sh

```diff
@@ -0,0 +1,2 @@
+cd $SRC_DIR
+py.test -vv
```

Comment:
I don't think `poliastro` is needed here, probably the scripts are inconsistent by accident. Will fix it.